sql/auth/sql_authorization.cc

```diff
@@ -227,6 +227,40 @@ opt_always_activate_roles_on_login is set to true.
 
  */
 
+const char *command_array[] = {"SELECT",
```

Comment:
Needs some static_assert that it's updated when new commands are added to the server